### ISSUE TITLE
Lack of screen reader support

### ISSUE THREAD

**Issue by veinfors** [qlik-oss/sn-list-objects#123]
We need to make sure screen readers works equally well as in the old filter pane.

(Might require keyboard navigation to be improved to be able to test this properly)

**Comment by johanlahti** [comment]
Current status: We support the most basic screen reader attributes, e.g. reading the label state ("A selected, B available" etc.) but we have to verify with a NVDA on a Win machine (unless there is a Mac version for it) and possibly fill some holes.

For instance, `aria-selected` seem not to work in Voice Over. Either, I'm missing something or this particular screen reader is not following the web standard.
